packages/onboarding/src/step-container/index.tsx

```diff
@@ -42,6 +42,8 @@ interface Props {
 	recordTracksEvent: ( eventName: string, eventProperties: object ) => void;
 	showHeaderJetpackPowered?: boolean;
 	showJetpackPowered?: boolean;
+	showHeaderWooCommercePowered?: boolean;
+	showWooCommercePowered?: boolean;
```

Comment:
Feel free to ignore this, but it wouldn't be clearer to name `showWooCommercePowered` something related to the footer, e.g: `showFooterWooCommercePowered`?